src/eval-frame/components/panes/console/app-message.jsx

```diff
@@ -26,9 +20,9 @@ const MESSAGES = {
 const AppMessage = ({ messageType }) => {
   const message = MESSAGES[messageType];
   return (
-    <AppMessageContainer>
+    <ConsoleMessage level="log">
```

Comment:
oops looks like this slipped thru, should be "LOG" now

Reply:
hmm makes me wonder if even on our purely declarative components we should include tests to make sure that they at least fully render without crashing (even tho such components don't have logic that needs testing). what do you think? like literally just a single test that makes sure shallow rendering doesn't crash would help us catch thing like this in the case of more complex presentational components with e.g. lots of children or props.

Reply:
I agree with this. I like these declarative bits but we still need to find a way of failing tests if someone doesn't declare properly.